server/tabby_server/services/resource_format.py

```diff
@@ -71,12 +74,22 @@ def google_books_test_call_api():
     return result_dict.output_dict, HTTPStatus.OK
 
 
+# Generic function for (hopefully) cleaner readability when it comes to
+# retrieving info about books.
+
+def get_book_attributes(book_list, book_attr):
```

Comment:
Since it's getting a single attribute, I think the name should be changed to `get_book_attribute()` or (`get_book_attr()` if you want for brevity.)

Reply:
Fair, I'll change it.